src/Microsoft.ML.PCA/PcaTransform.cs

```diff
@@ -98,40 +98,72 @@ public bool TryUnparse(StringBuilder sb)
             }
         }
 
+        public sealed class ColumnInfo
+        {
+            public readonly string Input;
+            public readonly string Output;
+            public readonly string WeightColumn;
+            public readonly int Rank;
+            public readonly int Oversampling;
+            public readonly bool Center;
+            public readonly int? Seed;
+
+            /// <summary>
+            /// Describes how the transformer handles one column pair.
+            /// </summary>
+            public ColumnInfo(string input,
```

Comment:
Can you copy comments about params from pigstension to here?

Reply:
done

---
In reply to: [227573714](https://github.com/dotnet/machinelearning/pull/1333#discussion_r227573714) [](ancestors = 227573714)